lib/honeybadger/json.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Honeybadger.JSON do
```

Comment:
Can we add a `@moduledoc false` to make it clear that we aren't documenting this one on purpose?